XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/OpenVR/OculusGoController.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
What's this? 

Did you take it from msft? I thought you wrote this yourself

Reply:
My Bad, it should be the XRTK license.  Good catch.